meta-lxatac-bsp/recipes-bsp/barebox/files/patches/series.inc

```diff
@@ -1,57 +1,32 @@
-# umpf-base: v2023.03.0
-# umpf-name: 2023.03.0/customers/lxa/tac
-# umpf-version: 2023.03.0/customers/lxa/tac/20230421-1
-# umpf-topic: v2023.03.0/topic/tlv
-# umpf-hashinfo: 38338204f9113c23c5d5f667cc209fe3a9476456
-# umpf-topic-range: e068bbf6a3608351484567af19da68e0e382fa0f..77abc3a4c4e8d2b240ac1edac15337d1132b0f19
+# umpf-base: v2023.10.0
+# umpf-name: 2023.10.0/customers/lxa/tac
+# umpf-version: 2023.10.0/customers/lxa/tac/20231024-1
+# umpf-topic: v2023.10.0/topic/tlv
+# umpf-hashinfo: dc2c1cf14ce0dceb10a10874d3e41f7aacbba6a4
+# umpf-topic-range: bc5e49335067d0c6c6e97c082a0368fe9d86b725..795783323c2d9c7890ba52382ba50e081a40e076
 SRC_URI += "\
   file://patches/0001-scripts-implement-slurp-a-read_file-with-fd-as-argum.patch \
   file://patches/0002-of_path-support-phandles-in-of_find_path.patch \
   file://patches/0003-Makefile-add-common-boards-include-to-include-search.patch \
   file://patches/0004-net-add-ethaddr-sequence-handling.patch \
-  file://patches/0005-common-add-global.serial_number-with-device-tree-fix.patch \
-  file://patches/0006-common-add-optional-systemd.hostname-generation.patch \
-  file://patches/0007-net-factor-out-eth_of_get_fixup_node.patch \
-  file://patches/0008-net-export-list-of-registered-ethernet-addresses.patch \
-  file://patches/0009-net-implement-ethaddr_string_cmp.patch \
-  file://patches/0010-common-add-barebox-TLV-support.patch \
-  file://patches/0011-commands-add-TLV-debugging-command.patch \
-  file://patches/0012-scripts-add-bareboxtlv-host-target-tool.patch \
+  file://patches/0005-common-add-optional-systemd.hostname-generation.patch \
+  file://patches/0006-net-factor-out-eth_of_get_fixup_node.patch \
+  file://patches/0007-net-export-list-of-registered-ethernet-addresses.patch \
+  file://patches/0008-net-implement-ethaddr_string_cmp.patch \
+  file://patches/0009-common-add-barebox-TLV-support.patch \
+  file://patches/0010-commands-add-TLV-debugging-command.patch \
+  file://patches/0011-scripts-add-bareboxtlv-host-target-tool.patch \
```

Comment:
If `umpf format-patch -n` was used the diff would be smaller and so easier to review.
(But I agree this is a subjective preference.)